bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TabsImpl.java

```diff
@@ -83,6 +85,30 @@ public String getAccessibilityLabel() {
         return accessibilityLabel;
     }
 
+    /*
+     * DataLayerProvider implementation of field getters
+     */
+
+    @Override
+    public String[] getDataLayerShownItems() {
+        String activeItemName = getActiveItem();
+        List<ListItem> items = getItems();
+        String activeItemId = null;
+
+        if (items != null) {
+            if (activeItemName == null) {
+                return new String[] { getItems().get(0).getId() };
+            }
+
+            activeItemId = items.stream()
+                .filter(e -> e.getName().equals(activeItemName))
+                .findFirst()
+                .get().getId();
```

Comment:
shouldn't it be instead?
`get().getDataLayer().getId() `

Reply:
Done in 2509408